content/guides/testing-strategies/working-with-graphql.md

```diff
@@ -115,6 +115,16 @@ context('Tests', () => {
 In the test below, the response is modified to test the UI for a single page of
 results.
 
+:::cypress-visit-mount-test-example
```

Comment:
In this example, the `cy.visit()` and `mount()` are so far away from the tabs, that I can't even see the change. I'm not sure the tabs add a lot of value here.

Reply:
Agree